onnx_chainer/functions/pooling.py

```diff
@@ -3,18 +3,28 @@
 
 def convert_AveragePooling2D(
         func, onnx_op_name, opset_version, input_names, output_names, parameters):
+    # Supports cover_all by setting extra padding
+    if func.cover_all:
+        dph, dpw = func.sy - 1, func.sx - 1
+    else:
+        dph, dpw = 0, 0
     if opset_version == 1 or opset_version == 7:
         return helper.make_node(
             onnx_op_name, input_names, output_names,
             kernel_shape=(func.kh, func.kw),
-            pads=(func.ph, func.pw, func.ph, func.pw),
-            strides=(func.sy, func.sx),
-            count_include_pad=1,
+            pads=(func.ph, func.pw, func.ph + dph, func.pw + dpw),
```

Comment:
@rezoo Sorry for the delay. When a Function which has `ksize=2`, `stride=2`, and `pad=1` with `cover_all=True` is given to this converter, `dph` and `dpw` will both be 1, and then `pads` for `make_node` function will be `(1, 1, 2, 2)`. This causes “Pad should be smaller than kernel” error when the converted ONNX operator is run with ONNXRuntime. This specification seems not to be written in the official document of ONNX, but you can confirm this by adding `pad=1` cases to the current tests.